### DISCUSSION TITLE
Slider

### DISCUSSION THREAD

**Discussion by annefranca** [Telefonica/mistica-design#1133] [Proposals]
### Goal
Olá
Para o componente Slider, vamos precisar alterar os tokens de cor, incluir o status Disabled, "remover" o Tooltip e a altura no nativo IOS.

### Context
Android
Token elemento Track

IOS
Token elemento Knob

### Impact
Proposta conceito Work & Co.

### Discussion Questions
Podemos alterar os tokens de cor desse componente?

Podemos incluir o status Disabled?

Na proposta da Work & Co, esse componente está na versão nativa mais recente, podemos propor essa alteração também?

### Do you have a visual proposal?
https://www.figma.com/file/JkA7OefLqga6XyfiqLq1dY/Vivo---Lib-Intermedi%C3%A1ria?node-id=364%3A22134&t=uvjO4aa61aIDO80J-1

**Comment by yceballost** [comment]
Olá Anne!

Por qué un elemento Knob es diferente en un slider y un switch?
![image](https://user-images.githubusercontent.com/6722153/225777407-625247aa-5d08-4ba7-b4f2-01391ff530fa.png)

No deberían de ser iguales?

**Reply by annefranca** [reply]
Bom dia, @yceballost :)

Nos alinhamos internamente e decidimos por usar o componente Slider (Android/IOS) 100% Mística sem alterações ou ajustes.
